go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/cmsgov/easi-app
 go 1.21
 
 require (
-	github.com/99designs/gqlgen v0.17.39
+	github.com/99designs/gqlgen v0.17.40
```

Comment:
Would you be able to upgrade this to the lastest version? Time of writing that's https://github.com/99designs/gqlgen/releases/tag/v0.17.43

Reply:
Yeah! Bumped it up. The releases look fairly innocent